web/index.php

```diff
@@ -1,8 +1,13 @@
 <?php
+require_once __DIR__ . '/../vendor/autoload.php';
 
-$url = @parse_url($_SERVER['REQUEST_URI'])['path'];
+use Pmi\Application\HpoApplication;
 
-if (preg_match("/^\/s(\/|$)/", $url)) {
+$app = new HpoApplication();
+$allowSymfony = !($app->isStable() || $app->isProd());
```

Comment:
This would be temporary, right? Thinking at a comment block above it might be useful.

Reply:
Comments added.